openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/ActivityFeedTab.component.tsx

```diff
@@ -420,6 +514,35 @@ export const ActivityFeedTab = ({
             ),
             key: 'tasks',
           },
+          ...(showAnnouncementsSubTab
+            ? [
+                {
+                  label: (
+                    <div
+                      className="d-flex justify-between"
+                      data-testid="announcement-sub-tab">
+                      <Space align="center" size="small">
+                        <AnnouncementsIcon
+                          style={COMMON_ICON_STYLES}
+                          {...ICON_DIMENSION}
+                        />
+                        <span>{t('label.announcement-plural')}</span>
+                      </Space>
+
+                      <span>
+                        {!isUserEntity &&
+                          getCountBadge(
+                            countData.data.conversationCount,
+                            '',
+                            activeTab === ActivityFeedTabs.ANNOUNCEMENTS
+                          )}
+                      </span>
+                    </div>
+                  ),
```

Comment:
I guess we have tabLabel component which can be used here

Reply:
yes, we have tabLabel component , but ig that's for the tabs ,not for the sub tabs. We can use this compo for sub tabs as well, but currently its not supporting icon display, which we require for sub tab.